src/main/java/org/cloudfoundry/community/servicebroker/s3/service/s3/S3.java

```diff
@@ -25,30 +26,21 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
-import com.amazonaws.services.s3.AmazonS3;
-import com.amazonaws.services.s3.model.Bucket;
-import com.amazonaws.services.s3.model.BucketTaggingConfiguration;
-import com.amazonaws.services.s3.model.ObjectListing;
-import com.amazonaws.services.s3.model.S3ObjectSummary;
-import com.amazonaws.services.s3.model.S3VersionSummary;
-import com.amazonaws.services.s3.model.TagSet;
-import com.amazonaws.services.s3.model.VersionListing;
-import com.google.common.collect.Lists;
+import java.util.List;
 
 /**
  * @author David Ehringer
  */
 @Component
 public class S3 {
-
     private static final Logger logger = LoggerFactory.getLogger(S3.class);
 
-    private final AmazonS3 s3;
+    private final AmazonS3 amazonS3;
```

Comment:
It looks like the only change in this file is renaming this instance variable. Is this necessary as part of this as it adds quite a bit of noise to the commit?